src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -31,6 +31,8 @@ export class GestureHandlerWebDelegate
     this.gestureHandler = handler;
     this.view = findNodeHandle(viewRef) as unknown as HTMLElement;
 
+    this.view.oncontextmenu = () => false;
```

Comment:
Do we want to disable the context menu for every handler? I think we could do it only when right mouse button is expected (and for long press on touch), no?

Reply:
From what I understood we wanted to disable it for every handler, but I don't mind doing it only when we expect right click.

Reply:
We can talk about it to make sure it will work consistently across platforms then